config/project/project.yaml

```diff
@@ -0,0 +1,76 @@
+dateModified: 1716932096
+email:
+  fromEmail: $SYSTEM_EMAIL_FROM
+  fromName: 'Viget Craft Starter'
+  replyToEmail: null
+  template: null
+  transportSettings:
+    host: $SYSTEM_EMAIL_HOSTNAME
+    password: $SYSTEM_EMAIL_PASSWORD
+    port: $SYSTEM_EMAIL_PORT
+    useAuthentication: '1'
+    username: $SYSTEM_EMAIL_USERNAME
+  transportType: craft\mail\transportadapters\Smtp
+meta:
+  __names__:
+    35b563a0-4662-40b9-b885-a8450a2868d9: 'Viget Craft Starter' # Viget Craft Starter
+    805d8826-faed-4186-9b88-f509eb9b07e6: 'Viget Craft Starter' # Viget Craft Starter
+    b7e66782-af96-4012-9e17-914134073ced: Simple # Simple
```

Comment:
It feels a little funny that these UUIDs will be repeated across projects... but it would be impossible for two Craft projects to be installed using the exact same DB tables, so it's not really an issue.